src/components/ActivityCollector/configValidators.js

```diff
@@ -13,7 +13,7 @@ governing permissions and limitations under the License.
 import { string, boolean } from "../../utils/validation";
 
 export default {
-  clickCollectionEnabled: boolean().default(true),
+  clickCollectionEnabled: boolean().default(false),
```

Comment:
I don't think we want to change the default yet. Changing the default is a breaking change. I documented a strategy for changing the default here: https://wiki.corp.adobe.com/display/DMSArchitecture/Migration+plan+to+collect+less+by+default
I think making the changes from the wiki should be a separate ticket/PR.

Reply:
I see, ok, I'll do that in a separate PR. Thank you Jon for referencing the Wiki